src/client/Client.cc

```diff
@@ -4222,6 +4228,7 @@ void Client::add_update_cap(Inode *in, MetaSession *mds_session, uint64_t cap_id
       if (&cap != in->auth_cap)
          ldout(cct, 0) << "WARNING: " <<  "inode " << *in << " caps on mds." << mds << " != auth_cap." << dendl;
 
+      ldout(cct, 0) << "HACK: cap.cap_id:" << cap.cap_id << " cap_id:" << cap_id << dendl;
       ceph_assert(cap.cap_id == cap_id);
```

Comment:
I don't have a good grasp of why with a single MDS and between restarts what is importing/exporting caps but I was hitting this _ceph_assert_. This log statement was to see the contents.